qa/vagrant/build.gradle

```diff
@@ -17,8 +19,23 @@
  * under the License.
  */
 
-apply plugin: 'elasticsearch.vagrantsupport'
-apply plugin: 'elasticsearch.vagrant'
+plugins {
+  id 'java'
+  id 'elasticsearch.build'
+  id 'elasticsearch.vagrantsupport'
+  id 'elasticsearch.vagrant'
+}
+
+dependencies {
+  compile "junit:junit:${versions.junit}"
+  compile "org.hamcrest:hamcrest-core:${versions.hamcrest}"
+
+  // needs to be on the classpath for JarHell
+  runtime project(':libs:elasticsearch-core')
```

Comment:
Shouldn't this just be testRuntime?

Reply:
Yeah that makes sense, but it's going to change to `compile` once we start adding tests because we're going to use some of the filesystem stuff in `IOUtils`